internal/backends/postgresql/postgresql.go

```diff
@@ -41,17 +41,26 @@ type stats struct {
 // collectionsStats returns statistics about tables and indexes for the given collections.
 //
 // If the list of collections is empty, then stats filled with zero values is returned.
-func collectionsStats(ctx context.Context, p *pgxpool.Pool, dbName string, list []*metadata.Collection) (*stats, error) {
-	if len(list) == 0 {
-		return new(stats), nil
-	}
-
+func collectionsStats(ctx context.Context, p *pgxpool.Pool, dbName string, list []*metadata.Collection, val bool) (*stats, error) {
```

Comment:
Why argument is called `val`?

Reply:
I wasn't able to think of a proper name for the bool arguement, so temporarily kept it as val (short for value). if you could provide a sugesstion, that would be great.

Reply:
`refresh`?